package.json

```diff
@@ -118,7 +118,6 @@
     "gettext-parser": "1.3.1",
     "gl-matrix": "^3.4.3",
     "invariant": "^2.2.4",
-    "ios-device-list": "1.1.37",
```

Comment:
I think there's also a script in package.json that'll reference this

Reply:
@ryan953, I'm not sure which script you're referring to. I didn't find any other usages of `ios-device-list`.

Reply:
oh sorry. what a cryptic message i left!

I looked over the `scripts: {..}` section and `"extract-ios-device-names": "ts-node scripts/extract-ios-device-names.ts"` looks like it should get zapped too